web/assets/scss/colors.scss

```
@@ -1,2 +1,11 @@
 $orange: #f37021;
 $grey: #63605d;
+
+//Bundle full header background color
+$bright-orange: #faa61a;
+
+//Bundle star color
+$dark-orange: #f15a22;
```

This is our primary color
```suggestion
$primary: #f15a22;
```